.all-contributorsrc

```diff
@@ -403,6 +403,15 @@
       "contributions": [
         "translation"
       ]
+    },
+    {
```

Comment:
please remove these codes. if this pr merged, we will use `all-contributors-bot` add you as a contributor.

Reply:
I have reverted those changes with 5629434